symbulate/distributions.py

```diff
@@ -270,15 +345,21 @@ def __init__(self, mean=0.0, var=1.0, sd=None):
         if sd is None:
             self.scale = np.sqrt(var)
         else:
-            self.scale = sd
-        
+            if sd > 0:
```

Comment:
Hi Professor, I'm slightly confused on what you mean or where the check should be placed?
Should I check in the else: if var > 0: then sd = np.sqrt(var) and else: var can't be less than 0?